### PR TITLE
Refactoring, reshaping, and a bit of refurbishing

### PR DESCRIPTION
**Pull request by amotl**
Also, a bit of yak shaving.

[image](https://github.com/crate/cratedb-guides/assets/453543/e7d76d00-8f5e-48ab-99b9-3cf2546060ba)

## Preview
https://cratedb-guides--9.org.readthedocs.build/

## Review » Content
**Content is still a bit thin on the inside,** so it needs to be expanded on behalf of subsequent iterations.

## Review » Style
Please advise about wording and icons. Choose from most of the Material Icons explorable at https://fonts.google.com/icons. [^1]

/cc @msbt, @matkuliak, @ckurze, @surister, @marijaselakovic, @karynzv, @proddata 

[^1]: Note that _some newer icons_ are not available within the CSS bundle of `sphinx-design` at the time it was built. So, please tell us two or more variants to make sure one of them will actually work.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+::::
+
+
+
+
+## Topics
+
+Native features of CrateDB paired with auxiliary software components provide
+optimal coverage of different application domains.
```
Comment (amotl) [inline docs/index.md:96]: Any idea about a different title for this section?

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+:link-type: ref
+:link-alt: Getting started with CrateDB
+:padding: 3
+:text-align: center
+:class-card: sd-pt-3
+:class-body: sd-fs-1
+:class-title: sd-fs-5
+
+{material-outlined}`rocket_launch;1.3em`
+:::
```
Comment (amotl) [inline docs/index.md:44]: Maybe the 🚀 is too much here? Do you have any other suggestions?

Icons can be selected from https://fonts.google.com/icons.

File: docs/index.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+
+::::
+
+
+
+## Reference Architectures
+
+Reference architectures involving
+CrateDB for various use-cases.
```
Comment (amotl) [inline docs/index.md:68]: Can someone find a better wording other than **involving**? Let's just use **using**?

File: docs/index.md (hunk 1 above)
Comment (marijaselakovic) [reply docs/index.md:96]: Application Domains?

File: docs/integrate/index.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+streamsets
+```
+
+
+:::{tip}
+Please also visit [Overview of CrateDB integration tutorials].
+:::
+
+
+[Overview of CrateDB integration tutorials]: https://community.cratedb.com/t/overview-of-cratedb-integration-tutorials/1015
```
Comment (marijaselakovic) [inline docs/integrate/index.md:29]: Here we can work on the categorization similar to:
Screenshot 2024-02-13 at 17 24 28

File: docs/index.md (hunk 2 above)
Comment (marijaselakovic) [reply docs/index.md:45]: Let's just make sure they are free to use :)

File: docs/index.md (hunk 1 above)
Comment (amotl) [reply docs/index.md]: Excellent. Updated.

File: docs/index.md (hunk 3 above)
Comment (amotl) [reply docs/index.md]: Thanks!

File: docs/index.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+(howtos)=
+(tutorials)=
+
+# CrateDB Guides and Tutorials
+
+CrateDB is a distributed and scalable SQL database for storing and analyzing
+massive amounts of data in near real-time, even with complex queries. It is
+PostgreSQL-compatible, and based on Lucene. 
+
+This section of the documentation guides you how to use CrateDB in practice.
```
Comment (amotl) [inline docs/index.md:12]: I will swap the order of those two paragraphs.

Comment (marijaselakovic) [reply docs/index.md]: Sounds good

**Review by marijaselakovic — APPROVED** [review]

File: docs/index.md (hunk 5 above)
Comment (amotl) [reply docs/index.md]: Now even shorter:

> About how to use CrateDB and CrateDB Cloud in practice.

/cc @matriv ⛑️

File: docs/integrate/index.md (hunk 4 above)
Comment (amotl) [reply docs/integrate/index.md:29]: Thanks. See...

- GH-10

**Comment by ckurze** [comment]
Is this public website indexed by Google already (/guide)? The navigation contains Self-Managed (Legacy) and Self-Managed (NG) - and if I click on it, the content jumps completely.

The menu needs an urgent cleanup. This is very irritating having it out there without a proper navigation. When I click on Self-Managed (NG) - I get redirected to Guides and Tutorials -> Installation. We need to find a diferent solution. Would it be possible to have the current "Guides and Tutorials" -> "Installation" as the top-level menu point "Self-Deployed Installation" or something similar?

**Comment by amotl** [comment]
/docs/guide is only semi-live. Nobody knows about it.

**Pull request merged**